### PR TITLE
Add descriptions to all published crates

### PR DESCRIPTION
**Pull request by ggwpez**
Missing descriptions (47):  

- [x] `cumulus/client/collator/Cargo.toml`
- [x] `cumulus/client/relay-chain-inprocess-interface/Cargo.toml`
- [x] `cumulus/client/cli/Cargo.toml`
- [x] `cumulus/client/service/Cargo.toml`
- [x] `cumulus/client/relay-chain-rpc-interface/Cargo.toml`
- [x] `cumulus/client/relay-chain-interface/Cargo.toml`
- [x] `cumulus/client/relay-chain-minimal-node/Cargo.toml`
- [x] `cumulus/parachains/pallets/parachain-info/Cargo.toml`
- [x] `cumulus/parachains/pallets/ping/Cargo.toml`
- [x] `cumulus/primitives/utility/Cargo.toml`
- [x] `cumulus/primitives/aura/Cargo.toml`
- [x] `cumulus/primitives/core/Cargo.toml`
- [x] `cumulus/primitives/parachain-inherent/Cargo.toml`
- [x] `cumulus/test/relay-sproof-builder/Cargo.toml`
- [x] `cumulus/pallets/xcmp-queue/Cargo.toml`
- [x] `cumulus/pallets/dmp-queue/Cargo.toml`
- [x] `cumulus/pallets/xcm/Cargo.toml`
- [x] `polkadot/erasure-coding/Cargo.toml`
- [x] `polkadot/statement-table/Cargo.toml`
- [x] `polkadot/primitives/Cargo.toml`
- [x] `polkadot/rpc/Cargo.toml`
- [x] `polkadot/node/service/Cargo.toml`
- [x] `polkadot/node/core/parachains-inherent/Cargo.toml`
- [x] `polkadot/node/core/approval-voting/Cargo.toml`
- [x] `polkadot/node/core/dispute-coordinator/Cargo.toml`
- [x] `polkadot/node/core/av-store/Cargo.toml`
- [x] `polkadot/node/core/chain-api/Cargo.toml`
- [x] `polkadot/node/core/prospective-parachains/Cargo.toml`
- [x] `polkadot/node/core/backing/Cargo.toml`
- [x] `polkadot/node/core/provisioner/Cargo.toml`
- [x] `polkadot/node/core/runtime-api/Cargo.toml`
- [x] `polkadot/node/core/bitfield-signing/Cargo.toml`
- [x] `polkadot/node/network/dispute-distribution/Cargo.toml`
- [x] `polkadot/node/network/bridge/Cargo.toml`
- [x] `polkadot/node/network/collator-protocol/Cargo.toml`
- [x] `polkadot/node/network/approval-distribution/Cargo.toml`
- [x] `polkadot/node/network/availability-distribution/Cargo.toml`
- [x] `polkadot/node/network/bitfield-distribution/Cargo.toml`
- [x] `polkadot/node/network/gossip-support/Cargo.toml`
- [x] `polkadot/node/network/availability-recovery/Cargo.toml`
- [x] `polkadot/node/collation-generation/Cargo.toml`
- [x] `polkadot/node/overseer/Cargo.toml`
- [x] `polkadot/runtime/parachains/Cargo.toml`
- [x] `polkadot/runtime/common/slot_range_helper/Cargo.toml`
- [x] `polkadot/runtime/metrics/Cargo.toml`
- [x] `polkadot/xcm/pallet-xcm-benchmarks/Cargo.toml`
- [x] `polkadot/utils/generate-bags/Cargo.toml`
- [x]  `substrate/bin/minimal/runtime/Cargo.toml`

### CODE REVIEW

**Comment by Morganamilo** [comment]
Bump. We now have CI for the crates.io stuff. Once this is fully done we can set the CI as required so crates never hit master without descriptions again.

Also as conflits are piling up maybe we want to merge this and do second round PR?

**Review by Morganamilo — APPROVED** [review]

**Review by skunert — APPROVED** [review]

**Comment by dmitry-markin** [comment]
@skunert Could you double-check descriptions for polkadot/node/network subsystems?

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4264875

File: cumulus/pallets/xcm/Cargo.toml [hunk 1]
```diff
@@ -4,6 +4,7 @@ edition.workspace = true
 name = "cumulus-pallet-xcm"
 version = "0.1.0"
 license = "Apache-2.0"
+description = "allet for stuff specific to parachains' usage of XCM"
```
Comment (joepetrowski) [inline cumulus/pallets/xcm/Cargo.toml:7]:
```suggestion
description = "Pallet for stuff specific to parachains' usage of XCM"
```

File: polkadot/node/core/backing/Cargo.toml [hunk 2]
```diff
@@ -4,6 +4,7 @@ version = "1.0.0"
 authors.workspace = true
 edition.workspace = true
 license.workspace = true
+description = "The Candidate Backing Subsystem. Tracks parachain candidates which can be backed, as well as the issuance of statements about candidates."
```
Comment (joepetrowski) [inline polkadot/node/core/backing/Cargo.toml:7]:
```suggestion
description = "The Candidate Backing Subsystem. Tracks parachain candidates that can be backed, as well as the issuance of statements about candidates."
```

File: polkadot/node/core/dispute-coordinator/Cargo.toml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 [package]
 name = "polkadot-node-core-dispute-coordinator"
 version = "1.0.0"
+description = "The node-side components which participate in disputes"
```
Comment (joepetrowski) [inline polkadot/node/core/dispute-coordinator/Cargo.toml:4]:
```suggestion
description = "The node-side components that participate in disputes"
```

File: cumulus/client/relay-chain-inprocess-interface/Cargo.toml [hunk 4]
```diff
@@ -3,6 +3,7 @@ authors.workspace = true
 name = "cumulus-relay-chain-inprocess-interface"
 version = "0.1.0"
 edition.workspace = true
+description = "Implementation of the RelayChainInterface trait for a polkadot full-node."
```
Comment (liamaharon) [inline cumulus/client/relay-chain-inprocess-interface/Cargo.toml:6]:
```suggestion
description = "Implementation of the RelayChainInterface trait for Polkadot full-nodes."
```

File: polkadot/node/network/bitfield-distribution/Cargo.toml [hunk 5]
```diff
@@ -1,6 +1,7 @@
 [package]
 name = "polkadot-availability-bitfield-distribution"
 version = "1.0.0"
+description = "Polkadot Bitfiled Distribution subsystem gossips signed availability bitfields used to compactly determine which backed candidates are available or not based on a 2/3+ quorum."
```
Comment (joepetrowski) [inline polkadot/node/network/bitfield-distribution/Cargo.toml:4]:
```suggestion
description = "Polkadot Bitfiled Distribution subsystem, which gossips signed availability bitfields used to compactly determine which backed candidates are available or not based on a 2/3+ quorum."
```

File: polkadot/node/network/collator-protocol/Cargo.toml [hunk 6]
```diff
@@ -1,6 +1,7 @@
 [package]
 name = "polkadot-collator-protocol"
 version = "1.0.0"
+description = "Polkadot Collator Protocol subsystem allows collators and validators to talk to each other."
```
Comment (joepetrowski) [inline polkadot/node/network/collator-protocol/Cargo.toml:4]:
```suggestion
description = "Polkadot Collator Protocol subsystem, which allows collators and validators to talk to each other."
```

File: polkadot/node/network/dispute-distribution/Cargo.toml [hunk 7]
```diff
@@ -1,6 +1,7 @@
 [package]
 name = "polkadot-dispute-distribution"
 version = "1.0.0"
+description = "Polkadot Dispute Distribution subsystem ensures all concerned validators are aware of a dispute and have the relevant votes."
```
Comment (joepetrowski) [inline polkadot/node/network/dispute-distribution/Cargo.toml:4]:
```suggestion
description = "Polkadot Dispute Distribution subsystem, which ensures all concerned validators are aware of a dispute and have the relevant votes."
```

File: cumulus/pallets/xcm/Cargo.toml (hunk 1 above)
Comment (liamaharon) [inline cumulus/pallets/xcm/Cargo.toml:7]:
```suggestion
description = "Pallet for parachain specific usage of XCM."
```

**Review by joepetrowski — APPROVED** [review]

File: polkadot/node/network/bitfield-distribution/Cargo.toml (hunk 5 above)
Comment (liamaharon) [inline polkadot/node/network/bitfield-distribution/Cargo.toml:4]:
```suggestion
description = "Polkadot Bitfiled Distribution subsystem. Gossips signed availability bitfields used to compactly determine which backed candidates are available or not based on a 2/3+ quorum."
```

File: polkadot/node/network/collator-protocol/Cargo.toml (hunk 6 above)
Comment (liamaharon) [inline polkadot/node/network/collator-protocol/Cargo.toml:4]:
```suggestion
description = "Polkadot Collator Protocol subsystem. Allows collators and validators to talk to each other."
```

File: polkadot/node/network/dispute-distribution/Cargo.toml (hunk 7 above)
Comment (liamaharon) [inline polkadot/node/network/dispute-distribution/Cargo.toml:4]:
```suggestion
description = "Polkadot Dispute Distribution subsystem. Ensures all concerned validators are aware of a dispute and have the relevant votes."
```

File: polkadot/node/network/gossip-support/Cargo.toml [hunk 8]
```diff
@@ -1,6 +1,7 @@
 [package]
 name = "polkadot-gossip-support"
 version = "1.0.0"
+description = "Polkadot Gossip Support subsystem is responsible for keeping track of session changes and issuing a connection request to the relevant validators on every new session."
```
Comment (liamaharon) [inline polkadot/node/network/gossip-support/Cargo.toml:4]:
```suggestion
description = "Polkadot Gossip Support subsystem. Responsible for keeping track of session changes and issuing a connection request to the relevant validators on every new session."
```

File: polkadot/node/service/Cargo.toml [hunk 9]
```diff
@@ -5,6 +5,7 @@ version = "1.0.0"
 authors.workspace = true
 edition.workspace = true
 license.workspace = true
+description = "Code that ties the different Polkadot components together and allows instantiation of a node."
```
Comment (liamaharon) [inline polkadot/node/service/Cargo.toml:8]:
```suggestion
description = "Utils to tie different Polkadot components together and allow instantiation of a node."
```

File: polkadot/runtime/metrics/Cargo.toml [hunk 10]
```diff
@@ -1,9 +1,11 @@
+
```
Comment (liamaharon) [inline polkadot/runtime/metrics/Cargo.toml:1]:
```suggestion
```

**Review by liamaharon — APPROVED** [review]

**Review by franciscoaguirre — APPROVED** [review]

**Review by gpestana — APPROVED** [review]

**Pull request merged**